profilepage/profilepage.js

```diff
@@ -0,0 +1,122 @@
+
+$document.ready(function() {
+  validateProfilePage();
+  $(".form-textfield input").blur(function)) {
+    var isBlank = $.trim($(this).val().length > 0;
+    $label = $(this).siblings(".form-label").toggleClass("form-label__floating",isBlank);
+
+  });
+});
+
+function validateProfilePage() {
+  $("#vitaProfileEdit").validate({
+    rules: {
+      "firstNameProfile":"required",
+      "lastNameProfile":"required",
+      email: {
+        required:true,
+        email: true
+      },
+      phone: {
+        required:true;
+      }
+    },
+    messages: {
+      email: {
+        required: "We need your email to confirm your shift times",
+        email: "Your email must be in the form of name@example.com"
+      }
+    }
+  });
+}
+function taxFunction() {
+	var taxValue = $("#taxSkills").val();
+	if (taxValue === "Yes") {
+		var myDiv = $("#skillType");
+		var array = ["international", "1040", "Visa"];
```

Comment:
So this is what I had mentioned in the general comments. Either we can have volunteers select these things and then send an email to a site admin to verify, or we can just have the site admins verify right off the bat (because a volunteer will need to send a site admin a paper copy of the certificate anyway, so there's no use in requiring a volunteer to do two things: say they can do it on our site and send the site admin their certificate). 

Additionally, these options will need to match up with things in the database when we actually do this stuff.